src/codegen/abi.js

```diff
@@ -8,6 +9,7 @@ const ABI = require('../abi')
 module.exports = class AbiCodeGenerator {
   constructor(abi) {
     this.abi = abi
+    this.name = path.basename(abi.name)
```

Comment:
This will break on Windows. It's been a rarity... but people use it. We may have to use regular string splitting here.